onnxruntime/core/session/custom_ops.cc

```diff
@@ -755,7 +756,10 @@ KernelCreateInfo CreateKernelCreateInfo(const std::string& domain, const OrtCust
   return KernelCreateInfo(def_builder.Build(), kernel_create_fn);
 }
 
-ONNX_NAMESPACE::OpSchema CreateSchema(const std::string& domain, const OrtCustomOp* op) {
+ONNX_NAMESPACE::OpSchema CreateSchema(const std::string& domain, const std::vector<const OrtCustomOp*>& ops) {
+  // The function registers the first schema assuming all the other one are the same except the types constraints.
+  ORT_ENFORCE(ops.size() > 0, "No kernels to registers.");
```

Comment:
this will probably trigger an abort in the new dummy extensions library that was required to resolve iOS build issues. we needed to create a stub RegisterCustomOps and it uses a build of extensions with no ops enabled to minimize the code change. Returning an error status may also break usage of that.

The registration isn't actually bad - it's just doing nothing, so logging a warning and exiting early would be more flexible.